tensornetwork/network_components.py

```diff
@@ -884,16 +929,49 @@ def _load_node(cls, node_data: h5py.Group) -> "CopyNode":
     name, shape, axis_names, backend = cls._load_node_data(node_data)
     copy_node_dtype = np.dtype(node_data['copy_node_dtype'][()])
     # pylint: disable=unnecessary-comprehension
-    node = CopyNode(
-        rank=len(shape),
-        dimension=shape[0],
-        name=name,
-        axis_names=[ax for ax in axis_names],
-        backend=backend,
-        dtype=copy_node_dtype)
+    node = CopyNode(rank=len(shape),
+                    dimension=shape[0],
+                    name=name,
+                    axis_names=[ax for ax in axis_names],
+                    backend=backend,
+                    dtype=copy_node_dtype)
 
     return node
 
+  def to_serial_dict(self) -> Dict:
+    """Return a serializable dict representing the node.
+    
+    Returns: A dict object.
+    """
+    node_dict = super().to_serial_dict()
+    node_dict.update({
+        'rank': self.rank,
+        'dimension': self.dimension,
+        'dtype': np.dtype(self.dtype).name,
+    })
+    if self.tensor is not None:
+      node_dict['tensor'] = self.backend.serialize_tensor(self.tensor)
+    return node_dict
+
+  @classmethod
+  def from_serial_dict(cls, serial_dict) -> "CopyNode":
+    """Return a node given a serialized dict representing it.
+    
+    Args:
+      serial_dict: A python dict representing a serialized node.
+      
+    Returns:
+      A node.
+    """
+    serial_dict['dtype'] = np.dtype(serial_dict['dtype']).type
+    tensor = serial_dict.pop('tensor', None)
+    node = cls(**serial_dict)
+
+    if tensor is not None:
+      node.tensor = backend_factory.get_backend(
+          serial_dict['backend']).deserialize_tensor(tensor)
+    return node
```

Comment:
another one

Reply:
Ah yes, thanks for catching those

Reply:
Go ahead a delete the copy node support. We'll be deleting that object anyway.